docs/source/tutorial/detection.rst

```diff
@@ -0,0 +1,176 @@
+:func:`chainercv.visualizations.vis_bbox` is a utility function that visualizes
+bounding boxes and an image together.
+
+
+Bounding Box Dataset
+--------------------
+ChainerCV supports dataset loaders, which can be used to easily index examples with list-like interface.
+Dataset classes whose names end with :obj:`BboxDataset` contain annotations of where objects locate in an image and which categories they are assigned to.
+These datasets can be indexed to return a tuple of an image, bounding boxes and labels.
+The labels are stored in an :obj:`np.int32` array of shape :math:`(R,)`. Each element corresponds to a label of an object in the corresponding bounding box.
+
+A mapping between an integer label and a category differs between datasets.
+This mapping can be obtained from objects whose names end with :obj:`label_names`, such as :obj:`chainercv.datasets.voc_bbox_label_names`.
+These mappings become helpful when bounding boxes need to be visualized with label names.
+In the next example, the interface of :obj:`BboxDataset` and the functionality of :obj:`vis_bbox` to visualize label names are illustrated.
+
+.. code-block:: python
+
+    from chainercv.visualizations import vis_bbox
+    from chainercv.datasets import VOCBboxDataset
+    from chainercv.datasets import voc_bbox_label_names
+    import matplotlib.pyplot as plt
+    dataset = VOCBboxDataset(year='2012')
+    img, bbox, label = dataset[0]
+    print(bbox.shape)  # (2, 4)
+    print(label.shape)  # (2,)
+    vis_bbox(img, bbox, label, label_names=voc_bbox_label_names)
+    plt.show()
+
+.. figure:: ../../image/detection_tutorial_bbox_dataset_vis.png 
+    :scale: 60%
+    :align: center
+
+Note that the example downloads VOC 2012 dataset at runtime when it is used for the first time on the machine.
+
+
+Detection Link
+--------------
+ChainerCV provides several network implementations that carry out object detection.
+For example, Single Shot MultiBox Detector (SSD) [Liu16]_ and Faster R-CNN [Ren15]_ are supported.
+Despite the difference between the models in how prediction is carried out internally,
+they support the common method for prediction called :meth:`predict`.
+This method takes a list of images and returns prediction result, which is a tuple of lists :obj:`bboxes, labels, scores`.
+The more description can be found here (:meth:`chainercv.links.SSD.predict`).
+Inference on these models runs smoothly by downloading necessary pre-trained weights from the internet automatically.
+
+.. code-block:: python
+
+    from chainercv.visualizations import vis_bbox
+    from chainercv.datasets import VOCBboxDataset
+    from chainercv.datasets import voc_bbox_label_names
+    from chainercv.links import SSD300
+    import matplotlib.pyplot as plt
+    dataset = VOCBboxDataset(year='2007', split='test')
+    img, _, _ = dataset[0]
+    model = SSD300(pretrained_model='voc0712')
+    bboxes, labels, scores = model.predict([img])
```

Comment:
`[img]` is a little bit tricky. Adding a comment like "make a list by [img] because predict takes a list of images" will be helpful. (I know you said the same thing in L. 87)